analyzer/protocol/openvpn/openvpn.evt

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
+protocol analyzer spicy::OpenVPN over UDP:
+	parse with OpenVPN::OpenVPNRecords;
+
+protocol analyzer spicy::OpenVPNHMAC over UDP:
```

Comment:
I'll test and make sure this works, then change.

Reply:
I think I already mentioned that - I also wonder if it might be worth it to combine the two TCP and the two UDP analyzers into one (so one UDP and one TCP analyzer). Which also gets rid of the big switch statements - and a lot of code repetition.

For TCP, you should be able to do the distinction of packet types by looking at the (provided) packet size. For UDP, you should be able to read the whole packet into into a bytes-member and ``&parse-from`` that one afterwards, passing the size along.

Reply:
I'm not exactly sure how to do this.

Reply:
But I made @rsmmr change.